src/richie-front/js/types/Course.ts

```diff
@@ -1,21 +1,14 @@
 import { Resource } from './Resource';
 
 export interface Course extends Resource {
-  end_date: string;
-  enrollment_end_date: string;
-  enrollment_start_date: string;
-  language: string;
-  organization_main: number;
+  start: string;
```

Comment:
The props should be alphabetically ordered

Reply:
why doesn't it break the build?

Reply:
I’m thinking this might not be linted because of declaration merging on interfaces.